app/helpers/discord_helper.rb

```diff
@@ -0,0 +1,15 @@
+module DiscordHelper
```

Comment:
It isn't a view helper. Inside services folder name files like:

```
- services
  - discord
    - helper.rb
    - service.rb
```

It'll be:

```ruby
module Discord
  module Helper
  ...
  end
end
```
and
```ruby
module Discord
  class Service
  ...
  end
end
```

Then edit places where are calling these (class and module) to the correct names:

```ruby
Discord::Service.new
include Discord::Helper
```